src/System.Linq/tests/OrderByTests.cs

```diff
@@ -441,5 +443,93 @@ public void TakeOne(IEnumerable<int> source)
             }
             Assert.Equal(1, count);
         }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void CultureOrderBy()
+        {
+            CultureInfo current = Thread.CurrentThread.CurrentCulture;
+            try
+            {
+                string[] source = new[] { "Apple0", "�ble0", "Apple1", "�ble1", "Apple2", "�ble2" };
+
+                string[] resultDK = new[] { "Apple0", "Apple1", "Apple2", "�ble0", "�ble1", "�ble2" };
+                string[] resultAU = new[] { "�ble0", "�ble1", "�ble2", "Apple0", "Apple1", "Apple2" };
+
+                string[] check;
+
+                CultureInfo dk = new CultureInfo("da-DK");
+                CultureInfo au = new CultureInfo("en-AU");
+
+                Thread.CurrentThread.CurrentCulture = dk;
```

Comment:
use CultureInfo.CurrentCulture.

Reply:
Ahh never knew that existed. I see that it only came in in Framework 4.6... [Docs say](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.currentculture?view=netframework-4.8) _"is a more performant equivalent"_, but when I look at the code (just ILSpy) the implementation is:

````
public static CultureInfo CurrentCulture
{
    get
    {
        return Thread.CurrentThread.CurrentCulture;
    }
}
````

(Not that it matters in this case, I am just wondering if this was a mistake?)

Reply:
> when I look at the code (just ILSpy) the implementation is

I think you're looking at .NET Framework.

Here's the implementation in .NET Core:
https://source.dot.net/#System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs,388